### ISSUE TITLE
Asset: Add sounds for various in-game actions

### ISSUE THREAD

**Issue by pauldpickell** [ubc-minetest-classroom/Minetest_Classroom#95]
- [x] "whoosh" sound when using a portal or teleporting to a new realm (as of [fbec7d1](https://github.com/ubc-minetest-classroom/Minetest_Classroom/commit/fbec7d1f4be838abffc33143faf5632566d5af32) and [be067d8](https://github.com/ubc-minetest-classroom/Minetest_Classroom/commit/be067d8fb8d777ce3f35d278f42b931ab6d0c62a))
- [x] page flip sound when using the plant compendium (as of [31fd491](https://github.com/ubc-minetest-classroom/Minetest_Classroom/commit/31fd4913af4b8150e7819f41713c4acaf74a1162) and [a2f9250](https://github.com/ubc-minetest-classroom/Minetest_Classroom/commit/a2f925072d4bdf371f46b90bf4690a82f93610a5))
- [x] chime/jingle when completing a tutorial
- [ ] information sound when a new task appears
- [x] background music
- [x] soundscapes around campus
- [x] notification sound on teacher receiving a report

**Comment by pauldpickell** [comment]
Could also extend this to music or other background soundscapes that can automatically play when you enter a realm or a building or a room.

**Comment by pauldpickell** [comment]
Might benefit from the geospatial audio project at UBC: https://eml.ubc.ca/projects/geospatial-audio/
@lukasdragon can we add the ability to register a sound file with a realm?

**Comment by lukasgolson** [comment]
> Might benefit from the geospatial audio project at UBC: https://eml.ubc.ca/projects/geospatial-audio/ @lukasdragon can we add the ability to register a sound file with a realm?

Sure thing! When we say register a sound file, would this correlate to something like background music / ambience?

**Comment by pauldpickell** [comment]
Yes, some sound file that would be played on a loop when the player is teleported to the realm. @mandyhuy we need to consider how to incorporate this into the UI.

**Comment by lukasgolson** [comment]
> Might benefit from the geospatial audio project at UBC: https://eml.ubc.ca/projects/geospatial-audio/ @lukasdragon can we add the ability to register a sound file with a realm?

Added in PR #154; See the test shack schematic for an example:
`/realm schematic load shack`
